js/script.js

```diff
@@ -0,0 +1,20 @@
+const taskInput = document.getElementById("task-input");
+const addButton = document.getElementById("btn-add");
+const unoderedList = document.getElementById("tasks");
+
+let tasks = [];
+const addTaskHandler = ()=>{
+    const taskTitle = taskInput.value.trim();
+    if(taskTitle){
+        tasks.unshift(taskTitle);
+        const tasksList = document.createElement("li");
+        tasksList.textContent = taskTitle;
+        unoderedList.prepend(tasksList);
+        taskInput.value = "";
+    }
+    else
+        alert("Please provide valid Task.");
+};
```

Comment:
It is recommended to not use DOM blocking events.

Reply:
I have used different approach now. Can you review it again? 
Thanks bhai.

https://github.com/sad-wtag/ToDoApp_VanillaJS/assets/164306406/f7094daf-6fee-4861-8c19-c419e74b4887